mempool/bench_test.go

```diff
@@ -46,6 +47,67 @@ func BenchmarkCheckTx(b *testing.B) {
 	}
 }
 
+func BenchmarkParallelCheckTx(b *testing.B) {
+	app := kvstore.NewApplication()
+	cc := proxy.NewLocalClientCreator(app)
+	mempool, cleanup := newMempoolWithApp(cc)
+	defer cleanup()
+
+	mempool.config.Size = 100000000
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	counter := make(chan int, 1000)
+	go func() {
```

Comment:
Why does this need to be in a goroutine?

Reply:
I've just always done the "channel full of ints" with a go routine and it means I don't need to know how many iterations the benchmarking tool is going to run beforehand. 

In any case, It doesn't change much, and I'm happy to take it out.

Reply:
no worries, just couldn't grasp why it was needed...